app/src/main/java/au/id/neasbey/biblecast/API/BibleAPIResponseParser.java

```diff
@@ -0,0 +1,22 @@
+package au.id.neasbey.biblecast.API;
+
+import android.text.Spanned;
+
+import java.util.List;
+
+/**
+ * Created by craigneasbey on 30/06/15.
+ *
+ * Bible API response parser utility
+ */
+public abstract class BibleAPIResponseParser {
+
+    /**
+     * Parse the response string
+     *
+     * @param responseString Response string to parse
+     * @param resultList Resulting list after parsing the string
+     * @throws BibleSearchAPIException
+     */
+    public abstract void parseResponseToList(String responseString, List<Spanned> resultList) throws BibleSearchAPIException;
```

Comment:
I think it would read better to return the List<Spanned> rather than pass it in?